CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 - Support for vm `KVM_SET_CLOCK`
 - Support for vm `KVM_GET_PIT2`
 - Support for vm `KVM_SET_PIT2`
+- Support for `KVM_GET_ONE_REG`
```

Comment:
Don't you need to `s/Unreleased/v0.3.0/` as well?

Reply:
True that. Will do tomorrow.

Reply:
Done.

Reply:
consistency nit:
```suggestion
- Support for vcpu `KVM_GET_ONE_REG`
```